packages/cubejs-schema-compiler/adapter/BaseMeasure.js

```diff
@@ -110,7 +110,7 @@ class BaseMeasure {
   }
 
   granularityFromInterval(interval) {
-    if (!interval) {
+    if (!interval || typeof interval === 'object') {
```

Comment:
This may lead to discrepancy problems. We either should provide a way to define it or to fallback to ungrouping here: https://github.com/cube-js/cube.js/blob/0744b132b15cd4c285bcc53b0d9f5916297f54c1/packages/cubejs-schema-compiler/adapter/BaseQuery.js#L417

Reply:
I will look through this and advise